gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -596,6 +638,20 @@ public void createBucket(String bucketName, CreateBucketOptions options) throws
   @Override
   public void createEmptyObject(StorageResourceId resourceId, CreateObjectOptions options)
       throws IOException {
+    Stopwatch stopwatch = Stopwatch.createStarted();
```

Comment:
Do we really need to measure each method, if we can measure HTTP requests latency as we do for Hadoop Metrics?

Reply:
Right now, we are not sure if we are spending time only a single HTTP calls or Batch/Higher level Op which involves multiple HTTP calls. This might be an overkill at this point. We will remove some/many of them after the debugging exercise 

I will be adding metrics for JSON apis at http layer as well.